.gitbook/5-oracles/1-operate-oracle-nodes/2-oracle-intialization.md

```diff
@@ -0,0 +1,133 @@
+```
+
+When run the above CLI for initializing the oracle, the `config.toml` file will be generated under
+the `/home_mnt/.oracle`
+in the enclave.
+The default `config.toml` file will be shown like this:
+
+```toml
+# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+# Write comment later
+###############################################################################
+###                           Base Configuration                            ###
+###############################################################################
+
+log-level = "info"
+oracle-mnemonic = ""
+oracle-acc-num = "0"
+oracle-acc-index = "0"
+data-dir = "data"
+
+oracle-priv-key-file = "oracle_priv_key.sealed"
+oracle-pub-key-file = "oracle_pub_key.json"
+node-priv-key-file = "node_priv_key.sealed"
+
+###############################################################################
+###                         Panacea Configuration                           ###
+###############################################################################
+
+[panacea]
+
+chain-id = ""
+grpc-addr = "http://127.0.0.1:9090"
+rpc-addr = "tcp://127.0.0.1:26657"
+default-gas-limit = "400000"
+default-fee-amount = "2000000umed"
+
+# A primary RPC address for light client verification
+
+light-client-primary-addr = "tcp://127.0.0.1:26657"
+
+# Witness addresses (comma-separated) for light client verification
+
+light-client-witness-addrs = "tcp://127.0.0.1:26657"
+
+# Setting log information for light client
+
+light-client-log-level = "error"
+
+###############################################################################
+###                         IPFS Configuration                           ###
+###############################################################################
+
+[ipfs]
+
+ipfs-node-addr = "127.0.0.1:5001"
+
+###############################################################################
+###                         API Configuration                           ###
+###############################################################################
+
+[api]
+
+listen-addr = "127.0.0.1:8080"
+write-timeout = "60"
+read-timeout = "15"
+```
+
+## Configuring Some Default Setting
+
+#### Base Configuration
+
+In `Base Configuration`, you need to configure a `oracle-mnemonic`, `oracle-acc-num`, and `oracle-acc-index`. Those
+components should be corresponded to account that you registered in the Panacea.
+
+#### Panacea Configuration
+
+In `Panacea Configuration`, you need to configure a chain ID of Panacea. If you want to join the oracle in the Panacea
+mainnet, the chain ID will be configured to `panacea-3`. Or if you want to join in the Panacea testnet, the chain ID
+will be `hygeia-8`.
+
+The default `grpc-addr` and `rpc-addr` setting is based on localnet. So if you want to connect with the Panacea
+mainnet, the `grpc-addr` will be `https://grpc.gopanacea.org` and the `rpc-addr` will be `https://rpc.gopanacea.org`.
```

Comment:
Would be better to recommend them to use the endpoints of their own node. Plus, you can mention that they can use the public endpoint provided by the Panacea team: https://github.com/medibloc/panacea-mainnet#public-endpoints

Reply:
Yes, It could be more better for the users.